docs/source/tutorials.rst

```diff
@@ -111,25 +84,52 @@ The `scxml_to_jani` tool takes an XML file, e.g. `main.xml <https://github.com/c
 All of those components are converted into one JANI DTMC model by the ``scxml_to_jani`` tool.
 
 
+Running the script
+`````````````````````
+
+After installing the AS2FM packages as described in the :ref:`installation section <installation>`, a full system model can be converted into a model-checkable JANI file as follows:
+
+.. code-block:: bash
+
+    cd AS2FM/jani_generator/test/_test_data/ros_example_w_bt/
+    scxml_to_jani main.xml
+
+The output is a JANI file will be located in the same folder, and it will be called `main.jani`.
+
+
+.. _jani_conversion:
+
+How to convert from CONVINCE robotic JANI to plain JANI?
```

Comment:
yeah .. maybe we can add a note that this is not really needed right now and completely independent of the ROS stuff